{
	"id": "openSUSE-SU-2024:14295-1",
	"modified": "2024-08-29T00:00:00Z",
	"published": "2024-08-29T00:00:00Z",
	"related": [
		"CVE-2024-0397",
		"CVE-2024-4030",
		"CVE-2024-4032",
		"CVE-2024-6923"
	],
	"summary": "python312-3.12.5-1.1 on GA media",
	"details": "These are all security issues fixed in the python312-3.12.5-1.1 package on the GA media of openSUSE Tumbleweed.",
	"affected": [
		{
			"package": {
				"ecosystem": "openSUSE:Tumbleweed",
				"name": "python312",
				"purl": "pkg:rpm/opensuse/python312&distro=openSUSE%20Tumbleweed"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "3.12.5-1.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"python312": "3.12.5-1.1",
						"python312-32bit": "3.12.5-1.1",
						"python312-curses": "3.12.5-1.1",
						"python312-dbm": "3.12.5-1.1",
						"python312-idle": "3.12.5-1.1",
						"python312-tk": "3.12.5-1.1",
						"python312-x86-64-v3": "3.12.5-1.1"
					}
				]
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY"
		},
		{
			"type": "WEB",
			"url": "https://www.suse.com/security/cve/CVE-2024-0397"
		},
		{
			"type": "WEB",
			"url": "https://www.suse.com/security/cve/CVE-2024-4030"
		},
		{
			"type": "WEB",
			"url": "https://www.suse.com/security/cve/CVE-2024-4032"
		},
		{
			"type": "WEB",
			"url": "https://www.suse.com/security/cve/CVE-2024-6923"
		}
	]
}
